src/SFA.DAS.Tools.Servicebus.Support.Web/Models/MessageListViewModel.cs

```diff
@@ -10,5 +10,6 @@ public class MessageListViewModel
         public QueueInfo QueueInfo { get; set; }
         public int Count { get; set; }
         public UserSession UserSession { get; set; }
+        public string GetQty { get; set; }
```

Comment:
Very pedantic, everywhere else we have property names which are not abbreviated. Can we change GetQty to be GetQuantity ? Not worried about Id's on HTML elements if that's your preference, but naming convention of C# properties should be explicit.

Reply:
removed